docs/rules/prefer-arrow-callback.md

```diff
@@ -1,77 +1,100 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
 
-Arrow functions are suited to callbacks, because:
+Arrow functions can be an attractive alternative to function expressions when describing callbacks or function arguments.
```

Comment:
Same comment as above. Maybe `Arrow functions can be an attractive alternative to function expressions for callbacks or function arguments.`?